pkgs/development/ruby-modules/rbenv/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, bash }:
+stdenv.mkDerivation rec {
+  pname = "rbenv";
+  version = "1.1.2";
+  
+  src = fetchFromGitHub {
+    owner = "rbenv";
+    repo = "rbenv";
+    rev = "v${version}";
+    sha256 = "12i050vs35iiblxga43zrj7xwbaisv3mq55y9ikagkr8pj1vmq53";
+  };
+
+  postPatch = ''
+     patchShebangs src/configure
+     cd src
+  '';
+
+  installPhase = ''
+  	mkdir -p $out/bin
+  	mv libexec $out
+  	mv completions $out
+  	ln -s $out/libexec/rbenv $out/bin/rbenv
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Groom your app’s Ruby environment";
+    longDescription = ''
+      Use rbenv to pick a Ruby version for your application and guarantee that your development environment matches production.
+      Put rbenv to work with Bundler for painless Ruby upgrades and bulletproof deployments.
+    '';
+    homepage = "https://github.com/rbenv/rbenv";
+    license = license.mit;
+    maintainers = with maintainers; [ fzakaria ];
```

Comment:
I will be in the maintainers starting with
https://github.com/NixOS/nixpkgs/pull/93365/files#diff-498fa90aaf17420b9719f0b04e1c3d08R9150-R9155